packages/showcase/src/app/components/accordion/accordion.component.html

```diff
@@ -1041,9 +1142,42 @@ <h2 id="other" class="heading-2">
           {{ accordionCustomHeader.item2.description }}
         </div>
 
-        <button class="button-secondary" slot="item-2-header-slot">
-          Button 2 Caption
-        </button>
+        <div
+          slot="item-2-header-slot"
+          class="custom-header-slot subtitle-semi-bold-xs"
+        >
+          Custom Header Caption 2
+          <div class="buttons-spacer">
+            <!-- TODO: remove buttons-spacer when segmented control is available -->
+            <button class="button-tertiary button-icon-only" [type]="'button'">
+              <ch-image
+                class="icon-md"
+                [src]="
+                  getIconPath({
+                    category: 'menus',
+                    name: 'paste',
+                    colorType: 'primary'
+                  })
+                "
+                [type]="'mask'"
+              ></ch-image>
+            </button>
+
+            <button class="button-tertiary button-icon-only" [type]="'button'">
+              <ch-image
+                class="icon-md"
+                [src]="
+                  getIconPath({
+                    category: 'menus',
+                    name: 'save',
+                    colorType: 'primary'
+                  })
```

Comment:
Avoid patterns calling a function in the template, since it will trigger the function call on each change detection life cycle.